lib/config/fileconf.go

```diff
@@ -2615,6 +2629,8 @@ type LDAPConfig struct {
 	DEREncodedCAFile string `yaml:"der_ca_file,omitempty"`
 	// PEMEncodedCACert is an optional PEM encoded CA cert to be used for verification (if InsecureSkipVerify is set to false).
 	PEMEncodedCACert string `yaml:"ldap_ca_cert,omitempty"`
+	// LocateServer is the config that enables LDAP server location using DNS SRV records.
+	LocateServer `yaml:"locate_server,omitempty"`
```

Comment:
```suggestion
	LocateServer `yaml:"locate_server"`
```
I don't think `omitempty` does anything for structs.